### DISCUSSION TITLE
Handle trusted circular contigs

### DISCUSSION THREAD

**Discussion by fplazaonate** [ablab/spades#907] [Q&A]
### Is your feature request related to a problem? Please describe. For generic questions use Q&A section in the Discussions forum above.

Hello,

I would like to perform hybrid metagenomic assembly combining Illumina and PacBio data.
I have generated a PacBio assembly with metaFlye. I would like to use it with the --trusted-contigs option. However, some contigs are circular but I don't know if  SPAdes can take this information into account?

**Comment by asl** [comment]
Your contigs are still linear in the input file. However, if the ends would overlap by at least largest k-mer length, then it would help a bit.

**Comment by fplazaonate** [comment]
It seems that there is no overlap at the ends. Is there any way to provide a GFA graph as input?

**Reply by asl** [reply]
No

**Comment by fplazaonate** [comment]
Ok. I will compare the PacBio assembly with the hybrid assembly and make sure that changes introduced in circular contigs are only indels or substitutions corrections.
